lib/experimental/modules/class-gutenberg-modules.php

```diff
@@ -90,27 +116,25 @@ public static function print_import_map() {
 	 * Prints all the enqueued modules using <script type="module">.
 	 */
 	public static function print_enqueued_modules() {
-		foreach ( self::$enqueued as $module_identifier ) {
-			if ( isset( self::$registered[ $module_identifier ] ) ) {
-				$module = self::$registered[ $module_identifier ];
-				wp_print_script_tag(
-					array(
-						'type' => 'module',
-						'src'  => $module['src'] . self::get_version_query_string( $module['version'] ),
-						'id'   => $module_identifier,
-					)
-				);
-			}
+		foreach ( self::get_enqueued() as $module_identifier => $module ) {
+			wp_print_script_tag(
+				array(
+					'type' => 'module',
+					'src'  => $module['src'] . self::get_version_query_string( $module['version'] ),
+					'id'   => $module_identifier,
+				)
+			);
 		}
 	}
 
 	/**
-	 * Prints the link tag with rel="modulepreload" for all the static
-	 * dependencies of the enqueued modules.
+	 * Prints the the static dependencies of the enqueued modules using <link rel="modulepreload">.
 	 */
-	public static function print_module_preloads() {
-		foreach ( self::get_dependencies( self::$enqueued, array( 'static' ) ) as $dependency_identifier => $module ) {
-				echo '<link rel="modulepreload" href="' . $module['src'] . self::get_version_query_string( $module['version'] ) . '" id="' . $dependency_identifier . '">';
+	public static function print_preloaded_modules() {
```

Comment:
the previous name was a small bit clearer to me, with "module preloads" instead of "preloaded modules." the previous version was communicating something about _the preloads_ themselves while this version changes the tense to the past and reads as if some modules have already been preloaded, which is not the case.

Reply:
Renamed back in https://github.com/WordPress/gutenberg/pull/57231/commits/8eb7356ccfb1c3544f94d7841523c6b352a6662c.